sage-ios/SAGE/SAGE/AddSchoolController.swift

```diff
@@ -41,15 +44,34 @@ class AddSchoolController: FormController {
         self.navigationController?.pushViewController(tableViewController, animated: true)
     }
     
+    func radiusButtonTapped() {
+        if self.location == nil {
+            self.showAlertControllerError("Please choose a location first.")
+        } else {
+            if self.location?.coordinate != nil {
```

Comment:
you know that self.location wont be nil at this point, so you dont need to make this nil check again